bundles/org.openhab.binding.bluetooth.bluegiga/src/main/java/org/openhab/binding/bluetooth/bluegiga/handler/BlueGigaBridgeHandler.java

```diff
@@ -341,7 +341,6 @@ public void bluegigaEventReceived(@Nullable BlueGigaResponse event) {
                 device = new BlueGigaBluetoothDevice(this, new BluetoothAddress(scanEvent.getSender()),
                         scanEvent.getAddressType());
                 devices.put(sender, device);
-                deviceDiscovered(device);
```

Comment:
I wouldn't remove this just yet. Discovery needs to be reworked here and I think some of that work is happening in PR #6921 . There is no harm in leaving this in for now.